mappings/net/minecraft/client/gui/widget/SliderButtonWidget.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS cxa net/minecraft/client/gui/widget/SliderButtonWidget
```

Comment:
Found this name quite confusing. What about something like `SliderControlWidget`?

Reply:
It is a "button" anyway.

Reply:
This is named based on `RealmsSliderButton`. We already have a `SliderWidget`, which is the superclass of `SliderButtonWidget`.